OIPA/iati/activity_manager.py

```diff
@@ -165,6 +165,35 @@ def prefetch_locations(self):
         from iati.models import LocationActivityDescription, Narrative, GeographicVocabulary
 
         # TODO: add correct narrative prefetches here - 2016-12-22
+        location_name_prefetch = Prefetch(
+            'name',
```

Comment:
i think the key here should be 'name__narratives' and similarly for the other narrative prefetches